junit-platform-gradle-plugin/src/main/groovy/org/junit/platform/gradle/plugin/EnginesExtension.groovy

```diff
@@ -39,5 +39,4 @@ class EnginesExtension {
 	void exclude(String... engineIds) {
 		this.exclude.addAll engineIds
 	}
-
```

Comment:
Is there a specific feature of the greclipse formatter that removes empty lines at the end of a block? The Java formatter does not enforce this. Can we turn it off?